.github/workflows/build.yml

```diff
@@ -33,4 +33,4 @@ jobs:
     - name: install dependencies
       run: sudo apt-get --quiet install ninja-build devscripts
     - name: build.py
-      run: ./src/build.py --sync-include=cmake,wabt --build-include=wabt,debian --no-test --no-host-clang
+      run: ./src/build.py --sync-include=cmake,wabt --build-include=wabt,debian --no-test --no-host-clang --no-sysroot
```

Comment:
Maybe add `v8` to sync-include instead?

Reply:
That will probably slow down this bot substantially. I guess the real question is, do we want to support truly-standalone use of this waterfall script or not?

Reply:
If we force use of the V8 repo with this script we can probably also remove the option to build without V8's clang.

Reply:
Would adding it to `sync` but not to `build` really slow it down that much?   I don't feel strongly about this.   Whichever you prefer.

Reply:
Yeah I don't know. probably not much, V8's DEPS aren't as bad as Chrome's. I don't really feel strongly either.
Since this is ready, I'll land it. If we want we can remove all of force_host_clang and sysroot-enabling changes together (which wouldn't really be any more work then removing force_host_clang by itself).